README.md

```diff
@@ -379,7 +379,9 @@ $ mvn package # Packages the application examples into a standalone jar
 
 | Branch (this repo)                      | Confluent Platform | Apache Kafka      |
 | ----------------------------------------|--------------------|-------------------|
-| [5.4.x](../../../tree/5.4.x/)\*         | 5.4.0-SNAPSHOT     | 2.4.0-SNAPSHOT    |
+| [5.4.0-post](../../../tree/5.4.0-post/) | 5.4.0              | 2.4.0             |
```

Comment:
There are many other versions missing... It is actually worth to list all bug fix releases?

Also 5.4.0-post does not exist yet -- should we do this change only after the release?

Reply:
Sounds reasonable. Thanks!

About bug-fix releases and what AK version: this is all documented in CP docs, hence, I am wondering if we should duplicate it here? Maybe better add a link to CP docs? It's already quite hard to keep the information up to date in one place.

Reply:
Should we also update older branches accordingly? (Maybe going back to 5.0.x?)